### DISCUSSION TITLE
Is there an option to only sign using the "sign & notarize" feature in v1.26?

### DISCUSSION THREAD

**Discussion by dehanj** [goreleaser/goreleaser#4884] [Q&A] [answered]
Hi, 
I'm using Gon, but are trying the new integrated feature [to sign macho binaries](https://goreleaser.com/customization/notarize/#notarize-macos-binaries) (from v1.26). What is driving me to use this instead, is the possibility to do it on any platform.

I wonder if there is a way to only use the signing feature? We are not using Notarization at the moment.

I have tried this using quill on the command line, and it works fine to only sign.
When I adapt my config to use it directly in Goreleaser it fails with this message:
```
  • sign & notarize macOS binaries
    • signing                                        binary=dist/darwin_universal_darwin_all/tkey-verification
    • sending notarize request                       binary=dist/darwin_universal_darwin_all/tkey-verification
    • took: 1s
  ⨯ release failed after 1s                  error=notarize: macos: dist/darwin_universal_darwin_all/tkey-verification: unable to parse EC private key for JWT: invalid key: Key must be a PEM encoded PKCS1 or PKCS8 key
```
The signing works, but I want to disable Notarization. Is that possible?

**Comment by caarlos0** [comment, marked as answer]
hey, it's not possible right now, definitely something to add though

**Reply by caarlos0** [reply]
opened https://github.com/goreleaser/goreleaser/issues/4890

**Reply by dehanj** [reply]
Thanks, appreciate it. Would be useful for us!

**Reply by caarlos0** [reply]
would something like #4919 do the trick?

**Reply by dehanj** [reply]
> would something like #4919 do the trick?

Looks promising, will comment directly in the PR!
